.github/workflows/create_sbom_report.yml

```diff
@@ -0,0 +1,119 @@
+  #Checking image version on available runner
+  version-check:
+    strategy:
+      matrix:
+        include:
+          - os: Windows
+            path: C:\
+          - os: macOS
+            path: /Users/runner/
+          - os: Linux
+            path: /imagegeneration/
+    defaults:
+      run:
+        shell: pwsh
+    needs: initialize
+    runs-on: ${{ needs.initialize.output.current-os }}
+    steps:
+      - name: Available image version check
+        run: |
+          if ("${{ runner.os }}" -eq "${{ matrix.os }}") {
+            if ((Get-Content ${{ matrix.path }}imagedata.json | jq '.[1] | .detail') -match '\d{8}\.+\d') { 
+              if ($Matches[0] -eq ${{ github.event.client_payload.imageLabel }}) {
+                echo "Current runner use ${{ github.event.client_payload.imageLabel }} image version."
+              } else {
+                echo "Error. Current runner image version don't match ${{ github.event.client_payload.imageLabel }} image version."
+                exit 1
+              }
+            } else {
+              echo "Cannot parse image version."
+              exit 1
+            }
+          }
+  #Install and run SYFT, compress SBOM, upload it to release assets
+  create-sbom:
+    strategy:
+      matrix:
+        include:
+          - os: Windows
+            path: D:/syft
+          - os: Linux
+            path: /usr/local/bin
+          - os: macOS
+            path: /usr/local/bin
+    defaults:
+      run:
+        shell: pwsh
+    needs: [initialize, version-check]
+    runs-on: ${{ initialize.output.current-os }}
+    steps:
+      - name: Install SYFT tool
+        run: |
+          echo "Trying to install SYFT tool from oficial sources"
+          if ("${{ runner.os }}" -eq "${{ matrix.os }}") {
+            echo "Installing SYFT for ${{ matrix.os }}"
+            curl -sSfL https://raw.githubusercontent.com/anchore/syft/main/install.sh | sh -s -- -b ${{ matrix.path }}
+          }
+      - name: Prepare SBOM report
+        run: |
+          if ("${{ runner.os }}" -eq "Windows") {
+            echo "Runnig SYFT tool on Windows on disk C:/"
+            D:/syft/syft dir:C:/ -vv -o spdx-json=SBOM.${{ github.event.client_payload.imageLabel }}.json
+          } elseif ("${{ runner.os }}" -eq "macOS") {
+              echo "Runnig SYFT tool on macOS with exludes"
+              syft dir:/ -vv -o spdx-json=SBOM.${{ github.event.client_payload.imageLabel }}.json --exclude ./Users --exclude ./System/Volumes --exclude ./private
+          } else {
+                echo "Runnig SYFT tool on Ubuntu with default settings"
+                syft dir:/ -vv -o spdx-json=SBOM.${{ github.event.client_payload.imageLabel }}.json
+          }
+          Compress-Archive sbom.${{ github.event.client_payload.imageLabel }}.json sbom.${{ github.event.client_payload.imageLabel }}.json.zip
+          echo "SBOM created successfully!"
+      #Might be changed to softprops/action-gh-release after additional check
```

Comment:
Looks like we don't have common logic between images so may be split it to 3 different steps?
```yaml
steps:
 - name: Prepare SBOM Windows
    if: ${{ runner.os == 'Windows' }}"
    run: |
        echo "Runnig SYFT tool on Windows on disk C:/"
        D:/syft/syft dir:C:/ -vv -o spdx-json=SBOM.${{ github.event.client_payload.imageLabel }}.json
 - name: Prepare SBOM macOS
    if: ${{ runner.os == 'macOS' }}"
    run: |
        echo "Runnig SYFT tool on macOS with exludes"
        syft dir:/ -vv -o spdx-json=SBOM.${{ github.event.client_payload.imageLabel }}.json --exclude ./Users --exclude ./System/Volumes --exclude ./private

```